scripts/export-bundles-sizes.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs')
+const path = require('path')
+const { execSync } = require('child_process')
+const { getOrg2ApiKey } = require('./lib/secrets')
+const { runMain, fetch: fetchWrapper } = require('./lib/execution-utils')
+
+const rumPath = path.join(__dirname, '../packages/rum/bundle/datadog-rum.js')
+const logsPath = path.join(__dirname, '../packages/logs/bundle/datadog-logs.js')
+const rumSlimPath = path.join(__dirname, '../packages/rum-slim/bundle/datadog-rum-slim.js')
+const workerPath = path.join(__dirname, '../packages/worker/bundle/worker.js')
+const versionPath = path.join(__dirname, '../packages/rum/package.json')
+
+const URL = 'https://http-intake.logs.datadoghq.com/api/v2/logs'
+const HEADERS = {
+  'DD-API-KEY': getOrg2ApiKey(),
+  'Content-Type': 'application/json',
+}
```

Comment:
**💬 suggestion:** ‏either inline those values in the function, or use more explicit names, like:

```
const LOG_INTAKE_URL =
const LOG_INTAKE_REQUEST_HEADERS =
```